README.md

```diff
@@ -55,3 +55,7 @@ sls plugin install -n serverless-python-requirements
 
 sls deploy --stage dev --region us-west-2
 ```
+
+## Data Lifecycle
+
+Drives are archived when there are no members automatically.  Deleted after 180 days.
```

Comment:
it says delete after 180day and there's a `GSUITE_DRIVER_DELETE` env var, but there's no delete code